scalasql/test/src/WorldSqlTests.scala

```diff
@@ -205,6 +205,54 @@ object WorldSqlTests extends TestSuite {
       // -DOCS
```

Comment:
`WorldSqlTests` is mostly the tutorial, which we don't need to specially call out multi-column `groupBy` in. Let's instead add these tests as cases in `scalasql.query.SelectTests groupBy`

Reply:
Should be fixed now